deriv_api/deriv_api.py

```diff
@@ -38,7 +37,8 @@
     'deriv_api.deriv_api.DerivAPI.disconnect': False,
     'deriv_api.deriv_api.DerivAPI.send': False,
     'deriv_api.deriv_api.DerivAPI.wsconnection': False,
-    'deriv_api.deriv_api.DerivAPI.storage': False
+    'deriv_api.deriv_api.DerivAPI.storage': False,
+    'deriv_api.deriv_api.DerivAPI._call_middleware': False
```

Comment:
Why the naming convention is different from others? Why not use `deriv_api.deriv_api.DerivAPI.middleware`

Reply:
Sorry this is the old name that should be deleted. 
will drop it.